scripts/prepdocs.py

```diff
@@ -177,8 +178,8 @@ def get_document_text(filename):
         for page_num, page in enumerate(form_recognizer_results.pages):
             tables_on_page = [
                 table
-                for table in form_recognizer_results.tables
-                if table.bounding_regions[0].page_number == page_num + 1
+                for table in (form_recognizer_results.tables or [])
```

Comment:
These were both optional attributes that we weren't treating as optional, so I put in some guards.